### PR TITLE
BSM Visualization Bugfix

### PR DESCRIPTION
**Pull request by drewjj**
# PR Details

## Description

This pull request resolves 2 bugs.

- The HTTP POST request to the CV Manager API BSM query endpoint was not properly serializing the object to JSON causing issues with parsing the file on the API. This has been resolved.
- The dropdown for the BSM visualization controller for the time steps was not displaying any options in the dropdown. This has been resolved.

I don't understand how these begun occurring because they should have never worked before this with how they needed to be resolved. The migration to typescript may have caused these issues. Not sure.

## How Has This Been Tested?

The unit tests have been updated and verified to run successfully. This has also been run in the local debug environment of the application with a port forwarded mongoDB from CDOT GCP.

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] My changes require new environment variables:
  - [ ] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
- [ ] My changes require updates to the documentation:
  - [ ] I have updated the documentation accordingly.
- [x] My changes require updates and/or additions to the unit tests:
  - [x] I have modified/added tests to cover my changes.
- [x] All existing tests pass.

### CODE REVIEW

File: webapp/src/pages/Map.tsx [hunk 1]
```diff
@@ -161,12 +161,21 @@ function MapPage(props: MapPageProps) {
...
-    { value: 15, label: '15 minutes', options: [] as number[] },
-    { value: 30, label: '30 minutes', options: [] as number[] },
-    { value: 60, label: '60 minutes', options: [] as number[] },
+    { value: 1, label: '1 minute' },
+    { value: 5, label: '5 minutes' },
+    { value: 15, label: '15 minutes' },
+    { value: 30, label: '30 minutes' },
+    { value: 60, label: '60 minutes' },
   ]
+  const [selectedOption, setSelectedOption] = useState({ value: 60, label: '60 minutes' })
```
Comment (mwodahl) [inline webapp/src/pages/Map.tsx]: Are we using this value anywhere? I noticed that it's not passed to our Select component anywhere

**Review by mwodahl — COMMENTED** [review]
Looks good when run locally & unit tests are passing - just one comment on Map.tsx

File: webapp/src/pages/Map.tsx (hunk 1 above)
Comment (drewjj) [reply webapp/src/pages/Map.tsx]: Good catch. This is leftover from a test that I needed to do to verify the issue with the selector dropdown options. I have now removed it.

**Review by mwodahl — APPROVED** [review]
Looks good!

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**